utilities/src/main/java/org/craftercms/commons/rest/BaseRestExceptionHandlers.java

```diff
@@ -43,15 +45,15 @@ protected ResponseEntity<Object> handleExceptionInternal(Exception ex, String me
     }
 
     @Override
-    protected ResponseEntity<Object> handleExceptionInternal(Exception ex, Object body, HttpHeaders headers, HttpStatus status,
-                                                             WebRequest request) {
-        logger.error("Request " + ((ServletWebRequest) request).getRequest().getRequestURI() + " failed with status " + status, ex);
+    protected ResponseEntity<Object> handleExceptionInternal(
+            Exception ex, @Nullable Object body, HttpHeaders headers, HttpStatusCode statusCode, WebRequest request) {
+        logger.error("Request " + ((ServletWebRequest) request).getRequest().getRequestURI() + " failed with status " + statusCode, ex);
```

Comment:
Switch to using the new style of logging with String Format, etc.

Reply:
Updated